app/src/main/java/com/example/mininpavelhw01/Handlers.kt

```diff
@@ -3,6 +3,13 @@ package com.example.mininpavelhw01
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 
+/**
+ * В данном файле реализованы классы и функции для разбора строки на операнды с использованием
+ * регулярных выражений. Данно решение позволяет без особых сложностей расширять функционал
+ * калькулятора, добавляя новый хэндлер в данном файле и добавляя его в PriorityQueue в файле
+ * StringExpressionAnalyzer
+ */
+
 val NUMBER_PATTERN: String
     get() = "(-?\\d+(\\.\\d+){0,1})"
```

Comment:
паттерн лучше положить в тот класс где он используется
в твоем случае так как это константа, то можно сделать так, без свойства 
```suggestion
private const val NUMBER_PATTERN: String = "(-?\\d+(\\.\\d+){0,1})"
```